e2e/web-gorilla-sqlz/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM golang:1.19-alpine
+COPY . /go/src/app
+WORKDIR /go/src/app
+CMD ./web-gorilla-sqlz
```

Comment:
It would better to remove the binary file, 
you should build web gorilla sqlz during the ci process